src/index.tsx

```diff
@@ -26,35 +26,59 @@ export type FeedbackData = {
   comment: string;
 }
 
-const Container = styled("div", {
+const Container = styled("form", {
   display: "flex",
   flexDirection: "column",
   alignItems: "center",
   maxWidth: "360px",
   position: "relative",
+  fontFamily: "sans-serif-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Cantarell, Ubuntu, roboto, noto, arial, sans-serif",
```

Comment:
we're having to repeat the font family in 4 places now, any way we can avoid that?